paasta_tools/marathon_tools.py

```diff
@@ -993,3 +1015,47 @@ def kill_task(client, app_id, task_id, scale):
             return []
         else:
             raise
+
+
+def get_instances_from_zookeeper(service, instance):
+    with ZookeeperPool() as zookeeper_client:
+        (instances, _) = zookeeper_client.get('/autoscaling/%s/%s/instances' % (service, instance))
+        return int(instances)
+
+
+def update_instances_for_marathon_service(service, instance, delta):
+    zookeeper_path = '/autoscaling/%s/%s/instances' % (service, instance)
+    marathon_config = load_marathon_service_config(
+        service=service,
+        instance=instance,
+        cluster=load_system_paasta_config().get_cluster(),
+        load_deployments=False,
+    )
+    with ZookeeperPool() as zookeeper_client:
+        instances = min(
```

Comment:
You might be jumping the gun a little bit on this function, but I understand why you wrote it.

It might be simpler if you had the function "just" take a number, and then make another function that does delta math. Don't go too nuts on this though, still early.